elasticsearch/standalone.compose

```diff
@@ -4,24 +4,19 @@ services:
     image: "docker.elastic.co/elasticsearch/elasticsearch:${version}"
     labels:
         com.datadoghq.sd.check.id: "elasticsearch"
-    container_name: elasticsearch1
+    ports:
+      - 9200
     environment:
       - cluster.name=docker-cluster
       - bootstrap.memory_lock=true
-      - network.host=__site__
-      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - network.host=_eth0_,_local_
+      - "ES_JAVA_OPTS=-Xms256m -Xmx256m"
     ulimits:
       memlock:
         soft: -1
         hard: -1
-    mem_limit: 1g
-    volumes:
```

Comment:
As we don't want to keep the data, no need to mount a volume.
Furthermore, these dangling volumes were a pain when restarting the containers